src/models/hooks/activityReportApprover.js

```diff
@@ -41,6 +41,12 @@ const updateReportStatus = async (sequelize, instance) => {
       submissionStatus: REPORT_STATUSES.SUBMITTED,
     },
   });
+
```

Comment:
if you look at the query above, I changed it from 
```js
  const report = await sequelize.models.ActivityReport.findByPk(instance.activityReportId, {
    attributes: ['submissionStatus', 'calculatedStatus'],
  });
```
to
```js
  const report = await sequelize.models.ActivityReport.findByPk(instance.activityReportId, {
    attributes: ['submissionStatus', 'calculatedStatus'],
    where: {
      submissionStatus: REPORT_STATUSES.SUBMITTED,
    },
  });
```
in order to be a little more succinct
However, it should bomb out early as before the rest of the function was locked in a 
```js
if(report.submittedStatus === REPORT_STATUSES.SUBMITTED){
/// stuff
}
```

and we don't want to update a report that hasn't been submitted by a user. I don't think this is causing a bug but it is how it was working before and I don't want to introduce any new ones if I can help it